### ISSUE TITLE
Platform receipt component - improvements

### ISSUE THREAD

**Issue by mrosvik** [Altinn/altinn-receipt#485]
Our platform receipt component (where you end up when navigating from archived elements in our message box) should use the new neutral design and should be optimized for smaller screens.

This is also breaking with WCAG: because of to low contrast on the white "x" button

- [ ] Add neutral design
- [ ] Optimize for smaller screens
